config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'users/show'
```

Comment:
Lines 2 and 3 can be deleted because `get "/:username" => "users#show", as: :user` already handles that action.